Pod/Classes/DatabaseConnection.swift

```diff
@@ -12,6 +12,8 @@ import sqlite3
 /** Valid SQLite types are marked using the 'SQLiteValue' protocol */
 public protocol SQLiteValue {}
 
+extension NSArray: SQLiteValue {}
+extension NSDictionary: SQLiteValue {}
```

Comment:
Do you think these should be defined in `SwiftyDB` instead? `TinySQLite` should not accept collections as it does not know how to handle store. It should probably be converted to NSData in `SwiftyDB` before being passed on